cli/index.js

```diff
@@ -45,6 +55,92 @@ vorpal
   })
 
 vorpal
-  .delimiter('🦄   $')
-  .show()
+  .command('eventstore create', 'A command line interface to eventstores')
+  .action((args, callback) => {
+    web3.eth.getAccounts( async (err, addresses) => {
+        if (err) { throw err }
+        if (args.options.from === undefined){
+          args.options.from = addresses[0]
+        }
+        let newAddress = await createEventStore(args.options.from);
+        console.log('🎁  ' + newAddress + ' EventStore created...')
+        console.log()
+        callback()
+    })
+  })
+
+  vorpal
+  .command('eventstore show ', 'A command line interface to eventstores')
+  .option('-f, --from <from>', 'from address')
+  .option('-c, --contractAddress <contractAddress>', 'contractAddress...')
+  .types({
+    string: ['contractAddress']
+  })
+  .action((args, callback) => {
+    web3.eth.getAccounts( async (err, addresses) => {
+        if (err) { throw err }
+        if (args.options.from === undefined){
+          args.options.from = addresses[0]
+        }
+        // console.log(args)
+        let bindingModel = {
+          contractAddress: args.options.contractAddress,
+          fromAddress: args.options.from
+        }
+        // console.log(bindingModel)
+        let readModel = await syncEventStore(bindingModel);
+              console.log(readModel)
+              console.log()
```

Comment:
This is a pretty awesome tool, but can you remove the extra console.logs in favor of `\n`